MIVisionX-setup.py

```diff
@@ -312,6 +312,12 @@
             os.system('sudo '+linuxFlag+' '+linuxSystemInstall +
                       ' '+linuxSystemInstall_check+' install -y miopen-hip migraphx')
 
+        #install MIGraphX
```

Comment:
@swetha097 can you please revert this change and sync with tot. This is not applicable to this PR